### ISSUE TITLE
Review connect to the database chapter

### ISSUE THREAD

**Issue by catalamarti** [oxford-pharmacoepi/OxinferOnboarding#5]
https://github.com/oxford-pharmacoepi/OxinferOnboarding/blob/main/connect_to_database.qmd

**Comment by nmercadeb** [comment]
My comments on the chapter:

- It might be helpful to include a brief introduction on why we connect from R to databases on a server. This could cover the "database side" and local R, and explain that while the database is in SQL, we work with it in R (a similar introduction to the one usually provided in CDMConnector courses)

- Overall, the beginning feels too strong, discussing hosts, ports, etc., without much prior context. Adding what I mention in the previous point could make it easier to follow.

- In the example with the cdm object, consider adding the prefix as it is mentioned in the text.

- Generally, the vocabulary is plain which I think is really helpful and makes it easier to follow. However, the sentence "this depends on the Database Management System (DBMS) of your back-end" in section 4.4. is too technical without prior explanation of what DBMS and back-end mean.

But overall having this will be really helpful to learn how to connect!!

**Comment by martaalcalde** [comment]
I agree with Nuria's comments, and completely second that this piece of work will be really helpful. I do have a couple of comments that might make things even clearer:
- Section 3.1 Databases: In the first sentence, there's a part that reads `(as of ?meta:date)`. I am not sure if this is a typo.
- Section 3.2 The servers: Mentioning the 100k subset here seems a bit out of context, as this sections focuses on servers. It might be clearer if this information was included in the next chapter (Connect to the database, 4.5 Database schemas). Currently, it seems like connecting to the db using the developers' servers will only allow access to the 100k subset.
